src/components/DocumentCard/DocumentCard.Props.ts

```diff
@@ -145,22 +146,10 @@ export interface IDocumentCardActionsProps extends React.Props<DocumentCardActio
   /**
    * The actions available for this document.
    */
-  actions: IDocumentCardAction[];
+  actions: IButtonProps[];
```

Comment:
BTW, this actually only breaks people who imports IDocumentCardAction.. anyone who specifies actions: { ... } without importing the type will not be broken...